src/social/components/UserInfo/UserMetadata.tsx

```diff
@@ -102,6 +107,40 @@ const MetadataEditModal = ({ user, onClose, isOpen, metadata, onSave }: Metadata
               {...register('isMigratedFromCircles')}
             />
           </Box>
+          <Box display="flex" flexDir="row" justifyContent="space-between">
+            <FormLabel>
+              <FormattedMessage id="userMetadata.localeLanguage" />
+            </FormLabel>
+            <RadioGroup
+              value={metadata?.localeLanguage?.[0]}
+              isInline={true}
+              colorScheme="primary"
+              onChange={(val) => setValue('localeLanguage.0', val as string)}
+              options={['en', 'de', 'es'].map((key) => ({
+                label: key,
+                value: key,
+              }))}
+            />
+          </Box>
+          <Box display="flex" flexDir="row" justifyContent="space-between">
+            <FormLabel>
+              <FormattedMessage id="userMetadata.businessType" />
+            </FormLabel>
+            <Select
+              options={['B2C', 'B2B'].map((key) => ({
```

Comment:
This is mostly an educational question, but for these types of enums do we usually just have them like this or do we define them in a more static/global sense like we do in the backend?

Reply:
So I _can_ pull in the actual proto enum and use that to populate it, but I also don't really want to give the options to do other things like UNKNOWN or TEST, so I figured I'd just hard code it. I'm open to differing opinions here though.

Reply:
FWIW, I _am_ copying the pattern seen on line 82 of this file.
It's probably best to pull them up to a const at the top of the file though, even if we plan on leaving them here and not pulling them from a proto enum.